src/actions.ts

```diff
@@ -18,29 +21,34 @@ interface Options {
   postCommentOnError?: boolean;
   settingsSchema?: TAnySchema;
   envSchema?: TAnySchema;
+  commandSchema?: TAnySchema;
   kernelPublicKey?: string;
+  disableSignatureVerification?: boolean; // only use for local development
```

Comment:
maybe change the comment to
```
/*
**
*/
```
and add a deprecation or a warning so the IDE warns the user when settings this variable.